### ISSUE TITLE
Crossword - marking of letters in grid relies solely on colour

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1081]
The crossword page currently uses green background on cells in the grid to indicate if they are correct. This needs to change so that the marking is also done in a non-colour dependant way. Possibly ticks/crosses after the clues when the whole word has been completed correctly. Although I this won't be simple to do as I think the the marking doesn't look at the whole word at the moment and just at individual input cells.

**Comment by ronm123** [comment]
@FayCross I don't have time to check right now but is it possible to add ticks just via css with :before or ;after like we did with gapfill etc e.g. https://github.com/thexerteproject/xerteonlinetoolkits/commit/559347ad7f3d6975c8cd206f840a0f9308647293

**Comment by FayCross** [comment]
The marking is only done on the crossword at the moment (not on clues) so it would need to be ticks & crosses on every individual letter in crossword. I think that would look quite messy. I will have a look at how other online crosswords do it

**Comment by JohnSmith-LT** [comment]
This probably needs someone to rework the css file as I've just hijacked the gapFill css section to add ticks and crosses on the :after pseudo element instead of the :before. Probably needs some more extensive testing also...

**Comment by FayCross** [comment]
@JohnSmith-LT I can see the ticks being added next to the clues after I submit my answers but they always seem to be ticks - even when the wrong answer is submitted

**Comment by JohnSmith-LT** [comment]
Probably something I'm overlooking for some fringe cases @FayCross - I only did a bit of testing but it was seeming to work fine:

![image](https://user-images.githubusercontent.com/3907362/166654355-0d85dc7c-9a08-49d7-bde7-23919948a733.png)

Can you share your example that doesn't work and i'll try to figure out where it's going wrong?

**Comment by JohnSmith-LT** [comment]
Is your Capitalisation on? Both of these have it on, image above was 3 attempts, image below was single attempt:

![image](https://user-images.githubusercontent.com/3907362/166655014-a4abbbaa-d6e4-426d-8755-18df43289d11.png)

**Comment by JohnSmith-LT** [comment]
In fact, scrap that, I also tried with mixed letters and it seems to work for me:

![image](https://user-images.githubusercontent.com/3907362/166655434-25ecb713-90ad-455c-b5b4-7801516fe497.png)

**Comment by FayCross** [comment]
I'm not changing any settings - just creating a crossword page & adding words to it. I've done a bit more testing and it's odd. See this example: https://xot310beta.dlearning.nl/play.php?template_id=66

Page 1 works:
![crossword1](https://user-images.githubusercontent.com/8926470/166656387-b113a20d-5dfc-46a9-9411-53d2b068a485.PNG)

Page 2 doesn't:
![crossword2](https://user-images.githubusercontent.com/8926470/166656407-4a9ef2d2-0bac-491f-b230-cf5975362483.PNG)

Same words for both and same mistakes entered (too for two and fine for five). Only difference between those pages is that the clues on page 1 match the answers while the clues on page 2 are numbers.

**Comment by JohnSmith-LT** [comment]
Ah right, that's my fault then @FayCross because I'm matching the words to the contents of the <p> tags in the Across/Down fields... didn't think about different words/clues... Not sure why they are all being considered correct though?!?

Leave that with me...

**Comment by JohnSmith-LT** [comment]
Try that latest commit @FayCross, seems to be working with those cases now:

![image](https://user-images.githubusercontent.com/3907362/166677395-cb7dc50d-f9e0-41f4-b15b-e6fadb4b59e3.png)

**Comment by FayCross** [comment]
@JohnSmith-LT thanks, that's working for me too now
